components/collections/choose-collection-type.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <div v-if="show">
+        <tabs style="m-4" :id="'collections_tab'" v-on:tab-changed="updateCollectionsType">
+            <tab
+                :id="'my-collections'"
+                :label="'My Collections'"
+                :selected="true"
+            >
+            </tab>
+            <tab
+                :id="'team-collections'"
+                :label="'Team Collections'"
+                :selected="false"
+            >
+                <ul>
+                    <li>
+                        <label for="select_team"> Select Team: </label>
+                        <span class="select-wrapper">
+                            <select 
+                            type="text" 
+                            id="team"
+                            class="team"
+                            v-model="selectedTeam"
+                            autofocus
+                            >
+                                <option
+                                v-for="(team, index) in myTeams"
+                                :key="index"
+                                :value="team"
+                                >
+                                {{ team.name }}
+                                </option>
+                            </select>
+                        </span>
+                    </li>
+                </ul>
+            </tab>
+        </tabs>
+
+    </div>
+</template>
+
+<script>
+import tabs from '../ui/tabs';
```

Comment:
All UI components are auto-imported. No need to import `tab` component explicitly. You can remove this LOC.